ramalama/cli.py

```diff
@@ -405,7 +416,7 @@ def _stop_container(args, name):
 
     conman_args = [conman, "stop", "-t=0"]
     if args.ignore:
-        conman_args += [ "--ignore", str(args.ignore)]
+        conman_args += ["--ignore", str(args.ignore)]
```

Comment:
Surprised flake8/black didn't pick this up

Reply:
I see some of our CI/CD system not working properly.  I have a failing that these are causing flake8/black to fail, but we are not catching the errors in ci/cd system